features/step_definitions/stream_steps.rb

```diff
@@ -14,6 +14,10 @@
   find(".toggle_nsfw_state", match: :first).click
 end
 
+When /^I toggle all nsfw posts$/ do
+  page.execute_script("$('a.toggle_nsfw_state').click()")
+end
```

Comment:
I think you are looking for [`all("...")`](http://www.rubydoc.info/github/jnicklas/capybara/master/Capybara/Node/Finders#all-instance_method)